docs/scalardb-supported-databases.md

```diff
@@ -140,3 +140,14 @@ For recommendations when using JDBC databases, see [Recommendations](requirement
 | **ScalarDB 3.9**  | ✅        |
 | **ScalarDB 3.8**  | ❌        |
 | **ScalarDB 3.7**  | ❌        |
+
+### YugabyteDB
+
+| Version           | YugabyteDB 2 |
+|:------------------|:-------------|
+| **ScalarDB 3.12** | ❌           |
```

Comment:
No released version supports this feature, so maybe we can remove this table?

Reply:
Let's keep this table. We can add a row for ScalarDB 3.13 when we release ScalarDB 3.13.